internal/backend/remote-state/azure/backend_test.go

```diff
@@ -127,6 +127,10 @@ func TestAccBackendOIDCBasic(t *testing.T) {
 }
 
 func TestAccBackendAzureADAuthBasic(t *testing.T) {
+	// cube2222: This doesn't seem to be a recommended mode of authentication and I have found no guide on how to configure it.
+	//           I've decided to skip this test in the suite, rather than spending more time to make it work.
+	t.Skip()
```

Comment:
Shouldn't this test be removed entirely rather than permaskipped? If it's not recommended, we could just not support it. If someone wants it, we can add it back later.

Reply:
I thought that this might be a better indication of intent, but I'm happy with removing it completely, too.

Reply:
I'll mention it in the commit message.

Reply:
Done.